docs/nftfair/users/activities.md

```diff
@@ -1,26 +1,31 @@
-# Homepage
+## Homepage
 
-The homepage is the main site where users can find a listing of past, current and future Fairdrop events. Every activity in the list contains information about a particular Fairdrop event, such as the network, the length of duration, the type of Fairdrop and the total amount of NFTs on sale (or to be given away). Users can click on the "Visit Activity Page" button to be redirected to the activity site, which is created specifically for an individual event. 
+The homepage is the main site where users can find a list of past, current and future Fairdrop events. Each activity contains information such as: 
+- The launch network
+- The event duration
+- The template format
+- The total number of NFTs
+ 
+Users can click on the "Visit Activity Page" button to be redirected to the activity site. 
 
----
+## Activity Sites
 
-# Activity Sites
+The activity site is where users participate in the Fairdrop event. To claim/purchase NFTs, users need to:
 
-The activity page is individually deployed and designated for a selected Fairdrop event. This user-facing site facilitates user interaction with the vault contract, as well as, providing detailed information about the Fairdrop, such as the remaining time, number of NFTs available to purchase, pricing and FAQs.
+- Be eligible for the Fairdrop based on the launch requirements 
+- Have a sufficient token balance to pay for the transaction fees and NFT
```

Comment:
Would this misleading users that they need native tokens such as eth or bnb to pay gas fee?